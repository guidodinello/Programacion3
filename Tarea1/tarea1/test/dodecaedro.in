leer_grafo
20 no-dirigido
1: 2 5 20 .
2: 1 3 18 .
3: 2 4 16 .
4: 3 5 14 .
5: 1 4 6 .
6: 5 7 13 .
7: 6 8 20 .
8: 7 9 12 .
9: 8 10 19 .
10: 9 11 17 .
11: 10 12 15 .
12: 8 11 13 .
13: 6 12 14 .
14: 4 13 15 .
15: 11 14 16 .
16: 3 15 17 .
17: 10 16 18 .
18: 2 17 19 .
19: 9 18 20 .
20: 1 7 19 .
0: .
cant_caminos 1
Fin
