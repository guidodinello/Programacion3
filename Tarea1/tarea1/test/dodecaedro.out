1>2>1 1 1 1 1 1 1 1 1 2 6 2 1 1 2 1 1 1 1 1 
3>Fin.
